packages/wdio-sumologic-reporter/src/index.ts

```diff
@@ -35,80 +48,78 @@ export default class SumoLogicReporter extends WDIOReporter {
         this.isSynchronising = false
```

Comment:
We can remove this if we tweak the property definition (see above)
```suggestion
```

Reply:
If we change it to false at line 26, the it is giving me an error stating that :
`A 'super' call must be the first statement in the constructor when a class contains initialized properties, parameter properties, or private identifiers.`